src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,52 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class DriveConstants {
+        // --> CAN IDs (motors)
+        public static final int kLeftDriveCAN1 = 0;
```

Comment:
Are these just placeholders for the real CAN IDs?

Reply:
These IDs need to be changed since we discovered running three motors under one CAN ID causes a hardware error.I can push a commit that numbers then 0-2 on one side and 3-5 on the other side.